src/api/model.js

```diff
@@ -34,4 +34,16 @@ app.post('/', (req, res) => {
         });
 });
 
+app.delete('/', (req, res) => {
+    addAction('DELETE_MODEL', {
+        modelAddress: req.query.modelAddress
```

Comment:
NACK: This failed when I tested the API. It needs to be `req.body` for `DELETE`s, not `req.query`. I'll fix this in a second.